README.md

```diff
@@ -143,9 +148,20 @@ configuration. Each target config may contain the following keys:
   `vm` and `web`.
   - If a target is not `web` compatible, it won't be compiled with the dart
     dev compiler, but that is the only effect of this attribute today.
+- **builders**: Optional, defaults to empty. The builders to apply to this
+  target. These are defined by this package or other packages in the `builders`
+  section of their bazelify.yaml.
+  - A list of either a Strings or Maps. If a Map is supplied then the key is the
```

Comment:
extra a in 'either a Strings'.

Does this accept a list of maps or a single map?